src/controllers/authController.js

```diff
@@ -88,7 +89,18 @@ const login = async (req, res) => {
   }
 };
 
+//logout endpoint
+
+const logout = async (req, res) => {
+  res.cookie('token', 'logout', {
+    httpOnly: true,
+    expires: new Date(Date.now() + 1000),
```

Comment:
We can set this to just `new Date()` (which would return just the current datetime). We don't need the logged out cookie to last for a specific amount of time.

Reply:
commit  d0963dfab2eba3423eacaf31d2876c72b05df7f2